### PR TITLE
removed the per-component max-change (gradient clipping) mechanism.

### PR DESCRIPTION
**Pull request by xiaohui-zhang**
We don't need the old max-change mechanism for nnet3 neural network training, since we've switched to a standard Frobenius norm based max-change mechanism done in nnet3/nnet-training.cc

### CODE REVIEW

**Comment by xiaohui-zhang** [comment]
ready to merge

**Comment by xiaohui-zhang** [comment]
ready to merge
